src/Examples/UnitModels/Operations/feed_testing.ipynb

```diff
@@ -0,0 +1,376 @@
+   "metadata": {},
+   "source": [
+    "In the following cell, we will be importing the necessary components from Pyomo and IDAES."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import objects from pyomo package \n",
+    "from pyomo.environ import ConcreteModel, SolverFactory, value\n",
+    "\n",
+    "# Import the main FlowsheetBlock from IDAES. The flowsheet block will contain the unit model\n",
+    "from idaes.core import FlowsheetBlock\n",
+    "\n",
+    "# Import the feed unit model\n",
+    "from idaes.models.unit_models import Feed\n",
+    "\n",
+    "# Import idaes logger to set output levels\n",
+    "import idaes.logger as idaeslog\n",
+    "\n",
+    "# Import the BTX_ideal property package to create a properties block for the flowsheet\n",
+    "from idaes.models.properties.activity_coeff_models import BTX_activity_coeff_VLE\n",
+    "\n",
+    "# Import the degrees_of_freedom function from the idaes.core.util.model_statistics package\n",
+    "# DOF = Number of Model Variables - Number of Model Constraints\n",
+    "from idaes.core.util.model_statistics import degrees_of_freedom\n",
+    "\n",
+    "# Create the ConcreteModel and the FlowsheetBlock objects, and attach the flowsheet block to it.\n",
+    "m = ConcreteModel()\n",
+    "\n",
+    "m.fs = FlowsheetBlock(dynamic=False) # dynamic or ss flowsheet needs to be specified here\n",
+    "\n",
+    "# Add properties parameter block to the flowsheet with specifications\n",
+    "m.fs.properties = BTX_activity_coeff_VLE.BTXParameterBlock(valid_phase='Liq',\n",
+    "                                                           activity_coeff_model=\"Ideal\")\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The initial degrees of freedom is: 3\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Create an instance of the feed unit, attaching it to the flowsheet\n",
+    "# Specify that the property package to be used with the feed is the one we created earlier\n",
+    "\n",
+    "m.fs.feed = Feed(property_package=m.fs.properties)\n",
+    "\n",
+    "# Call the degrees_of_freedom function, get initial DOF\n",
+    "DOF_initial = degrees_of_freedom(m)\n",
+    "print('The initial degrees of freedom is: {0}'.format(DOF_initial))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "tags": [
+     "testing",
+     "remove_cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "assert DOF_initial == 3"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The final degrees of freedom is: 0\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Fix the outlet conditions\n",
```

Comment:
1. This should be feed conditions rather than outlet conditions (as you are fixing things in the "feed" port).
2. I think there needs to be a little be of text explanation before this to indicate that feed blocks have the "feed" port, and what this means.